ui/src/AppCharting/AppChartC3.svelte

```diff
@@ -258,15 +251,22 @@
   }
 
   function styleSpecialPoints() {
-    styleDataPointErrors();
-    styleMissingPoints();
+    // Microtask Hack.
+    setTimeout(() => {
+      styleDataPointErrors();
+      styleMissingPoints();
+    }, 0);
   }
+
+  $: redrawKey = chartType || chartData || axis || Date.now();
 </script>
```

Comment:
Why is `Date.now()` included in this? It seems like that would make it always redraw because the time is always changing.

Reply:
Date.now() is just the redrawKey value.   Date.now() wouldn't be reactive only the result of any other property in this list being reactive via the `$:` subscriber